### PR TITLE
Add APIs for 4.7

### PR DESCRIPTION
**Pull request by jboxman**
Let's rock!

cc @vikram-redhat @lbarbeevargas @bobfuru 

API version changes

- RuntimeClass [node.k8s.io]: v1beta1 => v1
- VolumeSnapshot [snapshot.storage.k8s.io]: v1beta1 => v1
- VolumeSnapshotClass [snapshot.storage.k8s.io]: v1beta1 => v1
- VolumeSnapshotContent [snapshot.storage.k8s.io]: v1beta1 => v1

APIs added

- ConsoleQuickStart [console.openshift.io/v1]
- PodNetworkConnectivityCheck [controlplane.operator.openshift.io/v1alpha1]
- AlertmanagerConfig [monitoring.coreos.com/v1alpha1]
- UserOAuthAccessToken [oauth.openshift.io/v1]
- OperatorCondition [operators.coreos.com/v1]
- Operator [operators.coreos.com/v1]
- PodSecurityPolicy [policy/v1beta1]
- RangeAllocation [security.internal.openshift.io/v1]
- OverlappingRangeIPReservation [whereabouts.cni.cncf.io/v1alpha1]

### CODE REVIEW

File: rest_api/operator_apis/network-operator-openshift-io-v1.adoc [hunk 1]
```diff
@@ -582,7 +609,7 @@ Type::
 
 | `iptablesSyncPeriod`
 | `string`
-| The period that iptables rules are refreshed. Default: 30s
+| An internal kube-proxy parameter. In older releases of OCP, this sometimes needed to be adjusted in large clusters for performance reasons, but this is no longer necessary, and there is no reason to change this from the default value. Default: 30s
```
Comment (bobfuru) [inline rest_api/operator_apis/network-operator-openshift-io-v1.adoc:612]: Can we ask for `s/OCP/OpenShift Container Platform`?

**Review by bobfuru — COMMENTED** [review]
Just one comment, not sure if it can be addressed. LGTM!

File: rest_api/operator_apis/network-operator-openshift-io-v1.adoc (hunk 1 above)
Comment (jboxman) [reply rest_api/operator_apis/network-operator-openshift-io-v1.adoc:612]: @bobfuru, we can ask for anything we want, I'm sure. :)

While the CNO is different, most API text lives in `openshift/api` repo and can be freely modified, although it requires rebuilding stuff with go and I've never gotten it to work. So in theory we can create PRs for many of the changes we might want.

**Pull request merged**